src/main/java/sg/ncl/MainController.java

```diff
@@ -929,6 +933,7 @@ public String editAccountDetails(
             HttpSession session) throws WebServiceRuntimeException {
 
         boolean errorsFound = false;
+        String editPwdStatus = "editPwdStatus";
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2068)